shared/chat/conversation/info-panel/notifications/container.js

```diff
@@ -71,17 +91,14 @@ const mergeProps = (_, dispatchProps: DispatchProps, ownProps: OwnProps): _Props
   }
 }
 
-class _NotificationsWithKey extends React.PureComponent<_Props> {
```

Comment:
can just use a pure function for this and can use lifecycle if we're just doing this kind of 'load on mount'